radionice/pocetni/lekcije/006_pridjevi_imenski_predikat/006_pridjevi_imenski_predikat.tex

```diff
@@ -0,0 +1,247 @@
+	
+\underline{U rječničkoj formi}, い pridjevi uvijek završavaju na い, dok な i の pridjevi mogu završavati na bilo koji znak.
+
+\fukudai{い pridjevi}
+
+
+Naučimo neke korisne い pridjeve vezane uz boje:
+
+	\vspace{10pt}
+	\begin{tabular}{|l|l|}
+		\hline
+		しろい&bijelo\\\hline
+		くろい&crno\\\hline
+		くらい&tamno\\\hline
+		あかるい&svijetlo\\\hline
+		あおい&plavo, plavo-zeleno, zeleno(svjetlo na semaforu), nezrelo\\\hline
+		あかい&crveno\\\hline
+		きいろい&žuto\\\hline
+	\end{tabular}
+	
+\vspace{10pt}
+Neki korisni い pridjevi vezani uz oblike:
+	
+	\vspace{10pt}
+	\begin{tabular}{|l|l|l|l|}
+		\hline
+		おおきい&veliko&せまい&usko\\\hline
+		ちいさい&malo&ひろい&široko\\\hline
+		むずかしい&teško (potrebno je mnogo truda)&たかい&visoko\\\hline
+		おもい&teško (ima veliku težinu)&ひくい&nisko\\\hline
+		まるい&okruglo&ふかい&duboko\\\hline
+		しかくい&kvadratičasto&あさい&plitko\\\hline
+	\end{tabular}
+	
+	\vspace{10pt}
+	\begin{tabular}{|l|l|}
+		\hline
+		ふとい&debelo\\\hline
+		あつい&debelo\\\hline
+		ほそい&tanko\\\hline
+		うすい&tanko, slabo, polu-prozirno\\\hline
+	\end{tabular}
+	
+	\vspace{10pt}
+	ふとい i ほそい se koriste za većinu stvari, uglavnom cilindrične objekte i ljude, dok se あつい i うすい koriste za plosnate stvari i stvari koje su naslagane jedno na drugo.
+
+	
+	\vspace{10pt}
+	I šećer na kraju:
+	
+	\vspace{10pt}
+	\begin{tabular}{|l|l|}
+		\hline
+		かわいい&slatko (za osobe, čupave kućne ljubimce i ostale stvari od kojih možete dobiti infarkt)\\\hline
+		おもしろい&zanimljivo, zabavno\\\hline
+		すごい&super, odlično (za osobe, situacije, kada se nečemu divite), grozno\\\hline
+		うるさい&(nešto što mi) ide na živce, glasno\\\hline
+	\end{tabular}
+	\vspace{10pt}
+
+	\begin{reibun}
+		\rei{その　ねこ　は　\underline{くろい}。}{Ta mačka je crna.}
+		\rei{にほんご　の　べんきょう　は　\underline{むずかしい}。}{Učenje japanskog je teško.}
+		\rei{デスクトップ　ピーシー　は　\underline{おもい}。}{Desktop PC je težak.}
+	\end{reibun}
+	
+	
+	\vspace{10pt}
+	\begin{tabular}{|l||l|l|}
+		\hline
+		\multicolumn{3}{c}{Konjugacije い pridjeva}\\\hline
+		 &Pozitivno&Negativno\\\hline
+		Neprošlost&-&-(い)くない\\\hline
+		Prošlost&-(い)かった&-(い)くなかった\\\hline
+	\end{tabular}
+
+	\vspace{10pt}
+
+	\begin{reibun}
+		\rei{ほん　が　\underline{あつくなかった}。}{Knjiga nije bila debela.}
+		\rei{\underline{ひくい}　やま　は　\underline{むずかしくない}。}{Niska planina nije teška(za svladati).}
+		\rei{ライブ　が　\underline{おもしろかった}。}{Live je bio odličan.(Live koncert)}
+	\end{reibun}
+	
+\fukudai{な pridjevi}
+
+な pridjevi su različiti po tome što se drugačije konjugiraju.*
+Zovemo ih tako jer im se mora pridodati な kada se nalaze ispred imenice u neprošlom pozitivnom vremenu.
+
+Zanimljiva stvar kod njih je to što se u svim ostalim slučajevima konjugiraju kao imenice. To znači da kada rečenicu završavamo sa な pridjevom, da se umjesto nastavka な, pridodaje nastavak だ kao i kod imenica.\footnotemark[1] Time se sam pridjev pretvara u glagolski oblik.
```

Comment:
Time se sam pridjev pretvara u imenski predikat na isti način kao i imenica. Jer i jest imenica.